PYME/LMVis/importTextDialog.py

```diff
@@ -31,21 +31,26 @@ def isnumber(s):
     except:
         return False
     
-
-
-class ImportTextDialog(wx.Dialog):
+class ColumnMappingDialog(wx.Dialog):
     requiredVariables = {'x':'x position [nm]',
                         'y':'y position [nm]'}
     recommendedVariables = {'A':'amplitude of Gaussian',
                             't':'time [frames]',
                             'sig':'std. deviation of Gaussian [nm]',
                             'error_x':'fit error in x direction [nm]'}
     niceVariables = {'error_y':'fit error in y direction [nm]'}
+    colNames = []
```

Comment:
why the class level empty lists here?  Surely these should be instance variables? I'd think it should be enough to just assign to `self.colNames` and `self.dataLines` in the constructor  / `_parse_header()`. Will delete if no objections.

Reply:
I think I left this in here just so the class functions would all run without error if there was no derived class.  `CheckColNames`, for example, has a reference to self.colNames. It's not really important for them to be there. No objections to deleting.